generators/uscore-r4/generator.rb

```diff
@@ -10,6 +10,19 @@
 OUT_PATH = '../../lib/app/modules'
 RESOURCE_PATH = '../../resources/us_core_r4/'
 
+PROFILE_URIS = {
+  smoking_status: 'https://build.fhir.org/ig/HL7/US-Core-R4/StructureDefinition-us-core-smokingstatus.json',
+  diagnostic_report_lab: 'https://build.fhir.org/ig/HL7/US-Core-R4/StructureDefinition-us-core-diagnosticreport-lab.json',
+  diagnostic_report_note: 'https://build.fhir.org/ig/HL7/US-Core-R4/StructureDefinition-us-core-diagnosticreport-note.json',
+  observation_lab: 'https://build.fhir.org/ig/HL7/US-Core-R4/StructureDefinition-us-core-observation-lab.json',
+  pediatric_bmi_age: 'https://build.fhir.org/ig/HL7/US-Core-R4/StructureDefinition-pediatric-bmi-for-age.json',
+  pediatric_weight_height: 'https://build.fhir.org/ig/HL7/US-Core-R4/StructureDefinition-pediatric-weight-for-height.json'
+}.freeze
+
+def validation_profile_uri(sequence)
+  "Inferno::ValidationUtil::US_CORE_R4_URIS[:#{PROFILE_URIS.key(sequence[:profile])}]" if PROFILE_URIS.key(sequence[:profile])
```

Comment:
I think it's worth assigning `PROFILE_URIS.key(sequence[:profile])` to variable.

Reply:
fixed